input/docs/getting-started/index.md

```diff
@@ -3,5 +3,5 @@ Title: Getting Started
 To get started using Axiom, use one of the guides below based on your development enviornment.
 
 * [Visual Studio on Windows](getting-started-visualstudio)
-* [Visual Studio on Mac](getting-started-mac)
```

Comment:
Have you verified that the getting-started-visualstudio tutorial works on all platforms?

Reply:
Not really, it’s just something there though. I could put a Coming Soon pages. Visual studio Mac Linux is fairly different but similar. I’ll put Coming soon but a brief memo in case they want to try to install Vis Stu Mac Lin.

Reply:
Instead of "Coming soon"  how about "Help us out" kind of text. Something that basically says, if you need this page, write it and issue a PR ;)